src/assets/Keycloak/ocis-with-keycloak.md

```diff
@@ -0,0 +1,161 @@
+---
+title: keycloak Server
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: March 26, 2024
+tags: owncloud infinte scale, ocis, ocis-with-keycloak
+banner: https://blog.jankaritech.com/src/assets/keycloak/images/keycloak-with-ocis.png
+---
+
+[In previous blog](src/assets/keycloak/keycloak.md), we explored the topic of Keycloak. In this current post, our focus shifts to discussing the setup of OCIS with Keycloak.
```

Comment:
```suggestion
[In the previous blog](src/assets/keycloak/keycloak.md), we explored the general functionality of Keycloak. In this current post, I will show how to setup of [OCIS](link to ocis) with Keycloak.
```

Reply:
Do you expect people to know what ocis is?

Reply:
Why would someone want to do that?

Reply:
As a reader, I would be interested how I can setup Keycloak for my own project :)